### PR TITLE
virt_top.py: Enabled option to install the "virt-top" package , if missing. 

### PR DESCRIPTION
**Pull request by santwana**
This patch enables the test-case to install the "virt-top" package from the repo, before test-case execution.

It's ideal to initially check for the package installed and later the command present, before execution.

Also replaced the Autotest API's with Avocado API's.

Signed-off-by: Santwana Samantray <santwana@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/virt_cmd/virt_top.py [hunk 1]
```diff
@@ -27,7 +35,7 @@ def run(test, params, env):
 
     id_result = virsh.domid(vm_name)
     if id_result.exit_status:
-        raise error.TestNAError("Get domid failed.")
+        raise exceptions.TestNAError("Get domid failed.")
```
Comment (balamuruhans) [inline libvirt/tests/src/virt_cmd/virt_top.py]: it is `exceptions.TestSkipError`

File: libvirt/tests/src/virt_cmd/virt_top.py [hunk 2]
```diff
@@ -54,9 +62,9 @@ def run(test, params, env):
...
-            raise error.TestFail("Command virt-top exit successfully, but "
-                                 "domid is expected")
+            raise exceptions.TestFail("Command virt-top exit successfully, but"
+                                      "domid is expected")
     else:
         if cmd_result.exit_status != 2:
-            raise error.TestFail("Command virt-top exit successfully with"
-                                 "invalid option:%s\n", cmd_result.stdout)
+            raise exceptions.TestFail("Command virt-top exit successfully with"
+                                      "invalid option:%s\n", cmd_result.stdout)
```
Comment (balamuruhans) [inline libvirt/tests/src/virt_cmd/virt_top.py]:
```
raise exceptions.TestFail("Command virt-top exit successfully with"
                          "invalid option:%s\n" % cmd_result.stdout)
```

Comment (santwana) [inline libvirt/tests/src/virt_cmd/virt_top.py]: Made changes as suggested.

**Comment by balamuruhans** [comment]
@santwana, Thanks for the patch.

**Pull request merged**